packages/components/src/components/Box/Box.tsx

```diff
@@ -1,7 +1,7 @@
 import { ReactNode, HTMLAttributes } from 'react';
 import styled, { css } from 'styled-components';
-import { InputState } from '../../support/types';
 import { getInputStateBorderColor } from '../form/InputStyles';
+import { InputState } from '../form/inputTypes';
```

Comment:
Not sure about this one. In this case, it's maybe not just an input type. Should I make it more generic (but we are going to create another more global type) or duplicate InputState for this component? What do you think @dahaca?

Reply:
Honestly I am sure I removed it in the `Input` PR.. Maybe I did not push. Anyways, `Box` should not be using `InputState` of course, also `InputState` does not need the `success` state anymore, I believe.

Reply:
I created separate type for Box for now